### DISCUSSION TITLE
Is FUNDING.yaml up to date?

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [mrcjkb/rustaceanvim#635] [Q&A] [answered]
### Checked other resources

- [X] This is not a bug report or feature request (Please use [issues](https://github.com/mrcjkb/rustaceanvim/issues/new/choose) for those).
- [X] This is a question about rustaceanvim, not rust-analyzer or Neovim.
- [X] This is [not a question about autocompletion](https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#im-having-issues-with-autocompletion).
- [X] I have read the README.
- [X] I have read the [FAQ](https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#left_speech_bubble-faq).
- [X] I have searched for a solution using `:h rustaceanvim`.
- [X] I have searched [existing discussions](https://github.com/mrcjkb/rustaceanvim/discussions).
- [X] I have searched the existing issues/discussions of other plugins related to this question (if applicable).
- [X] I have searched the [existing rust-analyzer discussions](https://github.com/rust-lang/rust-analyzer/discussions) (if applicable).
- [X] I have added a very descriptive title to this question.

### Description

Is [FUNDING.yaml](https://github.com/mrcjkb/rustaceanvim/blob/master/.github/FUNDING.yml) up to date? Buy me a coffee and paypal point to the author of `rust-tools`, was it intentional?

### Example Code

```lua
.
```

### Neovim version (nvim -v)

.

### Operating system/version

.

### Output of :checkhealth rustaceanvim

```console
.
```

**Comment by mrcjkb** [comment, marked as answer]
Hey 😃 
Thanks for the donation.

Yes, as rustaceanvim started out as a fork of rust-tools (and probably wouldn't exist without it), I would like people to have the option to donate to simrat39.
